cmake.Modules/CompileFlags.cmake

```diff
@@ -24,7 +24,7 @@ IF(UNIX)
         SET(CMAKE_MACOSX_RPATH 1)
     ENDIF()
 
-    SET (CXX_OPTIONS       " -std=c++11 -Wno-deprecated -Wall -Werror -Wshadow -Wextra -Wfloat-equal -Wpointer-arith -Wwrite-strings -Wpacked")
+    SET (CXX_OPTIONS       " -std=c++11 -Wno-deprecated -Wall -Wextra -Wfloat-equal -Wpointer-arith -Wwrite-strings -Wpacked")

```

Comment:
I remember that I got a lot of warnings and couldn't compile with that setting, even before writing any code myself.
I'll try a fresh compile with Werror and will fix any warnings that might be left.